src/Directory.Build.props

```diff
@@ -0,0 +1,10 @@
+<Project>
+
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
```

Comment:
the second argument is unnecessary, GetPathOfFileAbove() will already start searching in the parent directory.